deploy/requirements.yml

```diff
@@ -41,4 +41,4 @@
 
 - name: opencraft
   src: https://github.com/open-craft/ansible-opencraft
-  version: 'origin/smarnach/opencraft'
+  version: 'origin/jbzdak/ssl-rebased'
```

Comment:
For now we plan to use commit hashes, but only for things that are merged to master, if you approve this one: https://github.com/open-craft/ansible-opencraft/pull/3 I'll update this to hash :)